src/OpenLoco/Graphics/Gfx.cpp

```diff
@@ -1924,20 +1803,4 @@ namespace OpenLoco::Gfx
     {
         _currentFontSpriteBase = base;
     }
-
-    static loco_global<Drawing::PaletteEntry[256], 0x0113ED20> _113ED20;
-
-    // 0x004523F4
-    void loadPalette()
```

Comment:
I don't think this should be moved it's for the actual rgb palette rather than the u8 palette maps